omnibus/package-scripts/inspec/postinst

```diff
@@ -17,7 +17,7 @@ error_exit()
 
 is_darwin()
 {
-  uname -v | grep "^Darwin" 2>&1 >/dev/null
+  uname -a | grep "^Darwin" 2>&1 >/dev/null
```

Comment:
Thanks for this! Would there be a case in which `uname -s` wouldn't work? (It seems that `-a` ensures that the `-s` value is supplied and `-v` is risky if the `-s` default is removed.